File: 0268-MissingNumber/Solution.java
import java.util.*;

public class Solution {
    
    static int missingNumber(int[] nums) {
        int n = nums.length;
        int totalSum = n;
        int inputSum = 0;
        for (int i = 0; i < n; i++) {
            totalSum += i;
            inputSum += nums[i];
        }
        return totalSum - inputSum;
    }

    public static void main(String[] args) {
        System.out.println();
    }

}

File: 0268-MissingNumber/solution.py
from functools import reduce
from operator import add


class Solution:

    def missing_number(self, nums: list[int]) -> int:
        n = len(nums)
        _all = n * (n + 1) // 2
        return _all - sum(nums)


if __name__ == "__main__":
    s = Solution()

